src/main/clm_varctl.F90

```diff
@@ -245,6 +245,12 @@ module clm_varctl
   integer, public :: CN_evergreen_phenology_opt = 0
   integer, public :: carbon_resp_opt = 0
 
+  !----------------------------------------------------------
+  ! Photosynthesis
+  !----------------------------------------------------------
+
+  integer, public :: itmax_hybrid = 40  ! max # of iterations used in subroutine hybrid that solves for ci and stomatal conductance
```

Comment:
This and the associated namelist read should be moved into PhotosynthesisMod, so that it can remain local to the module that uses it.